### PR TITLE
Add extra metadata attributes

### PR DESCRIPTION
**Pull request by Lenninlasd**
Migrate attributes properties in database

a46pqsovmb

Related issue: https://github.com/cga-harvard/worldmap/issues/69

### CODE REVIEW

**Comment by capooti** [comment]
Nice! does this work in GeoExplorer?

**Comment by Lenninlasd** [comment]
I haven't validated yet, how can I enable the gxp bar, in the map? currently I just see the map preview alone:

image

**Comment by capooti** [comment]
Not in the map within the layer details page.
But in any map containing the layer, when identifying a feature the attributes must be displayed using the information provided with the metadata (label, order, visible).

**Comment by waybarrios** [comment]
@capooti Is there anything else to do about this PR?

File: geonode/layers/models.py [hunk 1]
```diff
@@ -421,6 +421,18 @@ class Attribute(models.Model):
         default=True)
     display_order = models.IntegerField(_('display order'), help_text=_(
         'specifies the order in which attribute should be displayed in identify results'), default=1)
+    
+    searchable = models.BooleanField(
```
Comment (capooti) [inline geonode/layers/models.py]: this stuff must be outside of GeoNode. Let's create a WMLayer model in wm_exra with a OneToOneField field to Layer

File: geonode/layers/templates/layouts/panels.html [hunk 2]
```diff
@@ -527,6 +529,8 @@ <h4 class="modal-title" id="exampleModalLabel">Message box</h4>
                                           <td>{{form.attribute_label}}</td>
                                           <td>{{form.description}}</td>
                                           <td>{{form.display_order}}</td>
+                                          <td>{{form.visible}}</td>
```
Comment (capooti) [inline geonode/layers/templates/layouts/panels.html]: this will be form.wmlayer.searchable

File: geonode/layers/models.py (hunk 1 above)
Comment (waybarrios) [reply geonode/layers/models.py]: In my case, this field is necessary to work dataverse migrations because it's referred in a couples of columns.

**Comment by capooti** [comment]
@waybarrios @Lenninlasd I am merging this as layers.models.py is already forked. When we will have more time we will move customization to worldmap and use unforked geonode.

**Pull request merged**